File: Solutions/IQCare.Core/IQCare.PMTCT.Core/Models/HEI/PatientIPTWorkup.cs
using System;

namespace IQCare.PMTCT.Core.Models.HEI
{
    public class PatientIptWorkup
    {
        public int Id { get; set; }
        public int PatientMasterVisitId { get; set; }
        public int PatientId { get; set; }
        public bool YellowColouredUrine { get; set; }
        public bool Numbness { get; set; }
        public bool YellownessOfEyes { get; set; }
        public bool AbdominalTenderness { get; set; }
        public string LiverFunctionTests { get; set; }
        public bool DeleteFlag { get; set; }
        public int CreatedBy { get; set; }
        public DateTime CreateDate { get; set; }
        public bool StartIpt { get; set; }
        public DateTime IptStartDate { get; set; }
        public int IptRegimen { get; set; }
    }
}

File: Solutions/IQCare.Core/IQCare/frontend/app/pmtct/_models/hei/PatientIptWorkup.ts
export interface PatientIptWorkup {
    Id?: number;
    PatientMasterVisitId?: number;
    PatientId?: number;
    YellowColouredUrine?: number;
    Numbness?: number;
    YellownessOfEyes?: boolean;
    AbdominalTenderness?: boolean;
    LiverFunctionTests?: string;
    DeleteFlag?: boolean;
    CreatedBy?: number;
    CreateDate?: Date;
    StartIpt?: boolean;
    IptStartDate?: Date;
    IptRegimen?: number;
}
